src/censys/cloud_connectors/azure_connector/connector.py

```diff
@@ -186,10 +189,7 @@ def get_dns_records(self):
         try:
             zones = list(dns_client.zones.list())
         except HttpResponseError as error:
-            # TODO: Better error handling here
-            self.logger.error(
-                f"Failed to get Azure DNS records: {error.message}", exc_info=True
-            )
+            self.logger.error(f"Failed to get Azure DNS records: {error.message}")
```

Comment:
Was it intentional to strip the exc_info from this log?

Reply:
Oh nevermind... that was part of "Fix Azure DNS stack trace being shown to end users"